LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Call-Em-All
```

Comment:
@mbrookes, do you know if we should have Call-Em-All in this repo?

Reply:
I would see something close to Base UI:

```suggestion
Copyright (c) 2023 Material-UI SAS
```